tools/bazel.rc

```diff
@@ -12,3 +12,8 @@ build --strategy=AngularTemplateCompile=worker
 build --symlink_prefix=dist/
 
 test --test_output=errors
+
+################################
+# Temporary Settings for Ivy   #
```

Comment:
settings to avoid using Ivy?

Reply:
yes. angular bazel build looks for that define

Reply:
I mean, change the comment, ppl will think this means we are enabling Ivy in legacy mode